edk2toolext/base_abstract_invocable.py

```diff
@@ -45,6 +45,10 @@ def GetLoggingFolderRelativeToRoot(self):
         ''' Return a path to folder for log files '''
         raise NotImplementedError()
 
+    def InputParametersConfigured(self):
+        ''' Do any necessary work once all the input parameters are configured. '''
```

Comment:
This function is called once all the input parameters are collected and can be used to initialize environment...